Modules/System/WebDAV/src/Model/WebDAVContent.Table.al

```diff
@@ -0,0 +1,66 @@
+table 5678 "WebDAV Content"
+{
+    Access = Public;
+    DataClassification = SystemMetadata; // Data classification is SystemMetadata as the table is temporary
+    Caption = 'WebDAV Folder';
+    TableType = Temporary;
+    Extensible = false;
+
+    fields
+    {
+        field(1; "Entry No."; Integer)
+        {
+            Caption = 'Entry No.';
+            AutoIncrement = true;
+        }
+        field(10; Name; Text[250])
+        {
+            Caption = 'Title';
```

Comment:
There might be a very good reason that I missed, but why the different caption (compared to the name)?

Reply:
corrected